superset-frontend/src/SqlLab/types.ts

```diff
@@ -128,3 +130,70 @@ export type RootState = {
   messageToasts: toastState[];
   common: {};
 };
+
+export type ExploreRootState = {
```

Comment:
Would it make sense to move this into where the rest of the explore types are and then import it in?

Reply:
It would! Moved the type in [`this commit`](https://github.com/apache/superset/pull/19855/commits/ab8510e54db95c1c3cd640d6c47496bfb88b57bb).